### ISSUE TITLE
Cannot install in ruby 2.6.0 in Windows with rubygems

### ISSUE THREAD

**Issue by francesco-loreti** [msgpack/msgpack-ruby#159]
ERROR:  Error installing msgpack:
        The last version of msgpack (= 1.2.4) to support your Ruby & RubyGems was 1.2.4. Try installing it with `gem install msgpack -v 1.2.4`
        msgpack requires Ruby version >= 2.0, < 2.6. The current ruby version is 2.6.0.0.

**Comment by kou** [comment]
The restriction is caused by rake-compiler.
You'll be able to install msgpack gem by `gem install msgpack --platform ruby`.

**Comment by kou** [comment]
https://rubygems.org/gems/msgpack/versions/1.2.4-x64-mingw32 shows "Required Ruby Version:
< 2.6, >= 2.0".

**Comment by kou** [comment]
Or providing msgpack gem compiled with the latest rake-compile will solve this.

**Comment by tagomoris** [comment]
Ah, I see... it's about Windows environment.
I'll release a minor update later to solve it using latest rake-compiler.

**Comment by tagomoris** [comment]
@kou I created a new release using `rake-compiler` 1.0.7 (latest), but it creates a gem binary with `Ruby >= 2.0, < 2.6.DEV`. Is it correct & intentional?
https://rubygems.org/gems/msgpack/versions/1.2.5-x86-mingw32

**Comment by kou** [comment]
Ah, sorry. You need to use the latest rake-compiler-dock.

**Comment by kou** [comment]
https://rubygems.org/gems/rake-compiler-dock/versions/0.7.0 includes Ruby 2.6 support.

**Comment by tagomoris** [comment]
I retried with `rake-compiler-dock` 0.7.0, but it produces a metadata like this:
```
required_ruby_version: !ruby/object:Gem::Requirement
  requirements:
  - - ">="
    - !ruby/object:Gem::Version
      version: '2.0'
  - - "<"
    - !ruby/object:Gem::Version
      version: 2.6.dev
```

**Comment by kou** [comment]
We need to add `2.6.0` here: https://github.com/msgpack/msgpack-ruby/blob/master/Rakefile#L69
Or we can remove `RUBY_CC_VERSION` at all. The default `RUBY_CC_VERSION` includes `2.6.0`: https://github.com/rake-compiler/rake-compiler-dock/blob/master/Dockerfile.mri#L120

**Comment by tagomoris** [comment]
@kou Oops, I've forgotten about it. I'll add notifications to myself to update these 🙇 
Thank you!

**Comment by tagomoris** [comment]
Released `v1.2.6` which supports Ruby 2.6 on Windows.
https://rubygems.org/gems/msgpack/versions/1.2.6-x86-mingw32
https://rubygems.org/gems/msgpack/versions/1.2.6-x64-mingw32

@francesco-loreti Thank you for reporting!
